src/routes/forgotPassword.js

```diff
@@ -28,8 +28,10 @@ export default (apiServer) => {
       type: 'forgot-password',
       user: {
         _id: response.result.items[0]._id,
-        email: response.result.items[0].email,
-        accountId: response.result.items[0].accountId
+        email: response.result.items[0].email
```

Comment:
If you didn't need to modify the tests, that means, that you are not checking for the output. Can you please add these to your test cases?